### DISCUSSION TITLE
Clarification about Week 12

### DISCUSSION THREAD

**Discussion by alexacoon** [gui2de/ppol768-spring23#344] [General]
Hey Professors, 
We had a few questions:
-do we need to generate a DGP 
OR
-are we able to use the data set in folder 2 data (plus variables relevant to our project) 
OR
-do we need to generate a survey on SurveyCTO to run that we then evaluate with our check?

-Team 5

**Comment by darkmaterials** [comment]
Sorry, we figured it out - re-read the prompt, and saw that we can 'use an existing dataset', so we'll use one of the datasets in the folder.
